src/modules/Accordion/AccordionTitle.js

```diff
@@ -43,17 +50,29 @@ export default class AccordionTitle extends Component {
     parent: 'Accordion',
   }
 
+  static create = createShorthandFactory(AccordionTitle, content => ({ content }))
```

Comment:
Actually, AccordionTitle is defined in such scenario and has right value ([as shown in babel example](https://babeljs.io/docs/plugins/transform-class-properties/)). Nevertheless I've placed it after the class is created because there was some issue with code coverage not seeing the line as covered.